automated-actions/AWS_RISK_CREDENTIALS_EXPOSED/README.md

```diff
@@ -0,0 +1,173 @@
+
+AWS proactively monitors popular code repository sites for exposed AWS Identity and Access Management (IAM) access keys. On detection of an exposed IAM access key, AWS Health generates an AWS_RISK_CREDENTIALS_EXPOSED CloudWatch Event. In response to this event, an automated workflow deletes the exposed IAM Access Key, summarizes the recent API activity for the exposed key, and sends the summary message to an Amazon Simple Notification Service (SNS) Topic to notify the subscribers- all orchestrated by an AWS Step Functions state machine.
+
+This repository contains sample code for all the Lambda functions depicted in the diagram below as well as an AWS CloudFormation template for creating the functions and related resources.
+
+![screenshot for instruction](images/Architecture.png)
+
+### Walkthrough of the Architecture
+1. An IAM Access Key is inadvertently uploaded to one of the popular code repositories monitored by AWS.
+1. AWS Health detects the key, generating a AWS_RISK_CREDENTIALS_EXPOSED CloudWatch Event. AWS also temporarily restricts the API calls the key is able to make.
+1. A configured CloudWatch Events rule matches this event, triggering an execution of the `ExposedKey` state machine, which has the following sub-steps:
+    * Delete the exposed IAM Access Key Pair
+    * Summarize recent API activity for the user from CloudTrail
+    * Send security notification message containing summary to `SecurityNotificationTopic` SNS Topic.
+
+## Running the Example
+ 
+#### Option 1: Launch the CloudFormation Template in US East - N. Virginia (us-east-1) 
+The backend infrastructure can be deployed in US East - N. Virginia (us-east-1) using the provided CloudFormation template.
+Click **Launch Stack** to launch the template in the US East - N. Virginia (us-east-1) region in your account:
+
+[![Launch Stack into N. Virginia with CloudFormation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=AWSHealthCredsExposed&templateURL=https://s3.amazonaws.com/aws-health-tools-projects/AWS-RISK-CREDENTIALS-EXPOSED/risk_credentials_exposed.output.yaml)
+
+(In the last page of the wizard, make sure to:
+
+1. Click the checkboxes to give AWS CloudFormation permission to **"I acknowledge that AWS CloudFormation might create IAM resources"** and **"I acknowledge that AWS CloudFormation might create IAM resources with custom names"**
+1. Follow the instructions to **"Create Change Set"** 
+1. Click **"Execute"**
+)
+
+#### Option 2: Launch the CloudFormation Template Manually 
+
+If you would like to deploy the template manually, you need a S3 bucket in the target region, and then package the Lambda functions into that S3 bucket by using the `aws cloudformation package` utility.
+
```

Comment:
Fixed in this commit:

https://github.com/aws/aws-health-tools/pull/26/commits/02a8a0183057856e5092eff733bf8edacab76eb2